tests/PKSim.Tests/Core/DomainHelperForSpecs.cs

```diff
@@ -120,12 +120,13 @@ public static Individual CreateIndividual()
          return individual;
       }
 
-      public static IParameter ConstantParameterWithValue(double value)
+      public static IParameter ConstantParameterWithValue(double value=10, bool isDefault = false)
```

Comment:
Enhance the helper method to define a default value and a default state so that I do not have to repeat it all the time :)